OpenRA.Mods.Common/Traits/World/Selection.cs

```diff
@@ -237,5 +244,22 @@ public void Deserialize(World world, List<MiniYamlNode> data)
 				}
 			}
 		}
+
+		List<MiniYamlNode> IGameSaveTraitData.IssueTraitData(Actor self)
+		{
+			var nodes = new List<MiniYamlNode>()
+			{
+				new MiniYamlNode("Selection", "", Serialize())
+			};
+
+			return nodes;
+		}
+
+		void IGameSaveTraitData.ResolveTraitData(Actor self, List<MiniYamlNode> data)
```

Comment:
Likewise here - rename `public void Deserialize(World world, List<MiniYamlNode> data)` to `void IGameSaveTraitData.ResolveTraitData(Actor self, List<MiniYamlNode> data)` and change its uses of `world` to `self.World`.